ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -35,14 +35,18 @@
         </p>
       {{/if}}
       {{#if this.pathWhiteSpaceWarning}}
-        <div class="has-top-margin-m">
-          <AlertBanner
-            @type="warning"
-            @message="Your secret path contains whitespace. If this is desired, you'll need to encode it with %20 in API calls."
-            @marginTop={{true}}
-            data-test-whitespace-warning
-          />
-        </div>
+        <Hds::Alert
```

Comment:
Extra div and css didn't seem necessary
<img width="1070" alt="Screenshot 2023-06-13 at 4 14 18 PM" src="https://github.com/hashicorp/vault/assets/68122737/09f9193c-0dd2-4d44-ad59-f4013a5cea33">